android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -25,7 +25,6 @@ import com.facebook.react.uimanager.events.RCTEventEmitter
 import kotlin.reflect.KClass
 
 private val EVENT_CLASS_TO_REACT_NATIVE_NAME_MAPPING = mapOf(
-    PlayerEvent::class to "event",
```

Comment:
This did nothing, so I removed.

Reply:
This is a bit of a flaw in our event emitter. From a typing perspective one would expect that this works - but it doesn't.

Reply:
Thanks for the clarification! It confused me too!